cmake/Modules/Packages/KSPACE.cmake

```diff
@@ -46,6 +46,42 @@ else()
   target_compile_definitions(lammps PRIVATE -DFFT_KISS)
 endif()
 
+option(LMP_HEFFTE  "Use heFFTe as the distributed FFT engine, overrides the FFT option."  OFF)
```

Comment:
Maybe not call this `LMP_HEFFTE` , since we don't use `LMP_` anywhere else.

Reply:
It was a suggestion from here:

https://github.com/lammps/lammps/pull/3639#issuecomment-1430103475

What is the lammps naming convention?

Reply:
CMake options are usually grouped by their purpose. In this case, since it only affects the FFT, I would suggest something like  `FFT_USE_HEFFTE`.

Reply:
Also, if read this right the suggestion was for the name of define, not the option itself.

Reply:
I see your point. In all my project, I always keep the same names for the defines and the CMake options, didn't expect lammps would do it differently.

I updated the name.